README.md

```diff
@@ -86,17 +86,21 @@ your_profile_name:
       username: your_mysql_username
       password: your_mysql_password
       ssl_disabled: True
+      charset: utf8mb4
+      collation: utf8mb4_0900_ai_ci
 ```
 
 | Option          | Description                                                                         | Required?                                                          | Example                                        |
 | --------------- | ----------------------------------------------------------------------------------- | ------------------------------------------------------------------ | ---------------------------------------------- |
-| type            | The specific adapter to use                                                         | Required                                                           | `mysql`, `mysql5` or `mariadb`                            |
+| type            | The specific adapter to use                                                         | Required                                                           | `mysql`, `mysql5` or `mariadb`                 |
```

Comment:
I take it that this change was not intentional?
```suggestion
| type            | The specific adapter to use                                                         | Required                                                           | `mysql`, `mysql5` or `mariadb`                            |
```

Reply:
@mwallace582, it was intentional, apologies about not being explicit about it. It's just making the table aligned before rendering markdown.

Before:
<img width="1011" alt="image" src="https://github.com/dbeatty10/dbt-mysql/assets/65259263/979b6161-6f5e-4ab9-8337-12a79811570f">

After:
<img width="938" alt="image" src="https://github.com/dbeatty10/dbt-mysql/assets/65259263/23d5a591-3fd2-4ab6-8811-2ab26efa945e">

Reply:
Ah yes I see. Thanks for fixing this!